src/main/java/codeu/model/data/User.java

```diff
@@ -58,4 +61,16 @@ public String getName() {
   public String getPasswordHash() {
     return passwordHash;
   }
+
+  /** Returns the admin status of this User. */
+  public Boolean getAdmin() {
+    if (getName().equals("admin"))
+        return true;
+    else return admin;
+  }
+
+  /** Gives the User admin status. */
+  public void giveAdmin() {
```

Comment:
`getAdminStatus` would be a better name, since `giveAdmin` feels like you are returning the ID of the admin. Here, and everywhere (including tests) where you have called it `admin` :-)